release_elemental.sh

```diff
@@ -11,10 +11,17 @@ bazel_root="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 deploy_target='@com_google_jsinterop_base//:deploy'
 license_header="$(bazel info output_base)/external/com_google_jsinterop_base/license.txt"
-group_id="com.google.elemental2"
+# The group id can be customized in case you want to build using a custom groupid
+group_id=${ELEMENTAL2_GROUP_ID:-com.google.elemental2}
 
-echo "enter your gpg passphrase:"
-read -s gpg_passphrase
+# The invoker of script can pass GPG password to avoid having
+# the script wait for user to supply credentials interactively
+gpg_passphrase=${ELEMENTAL2_GPG_PASS:-}
```

Comment:
I think having the GPG password in a variable is a bad idea security-wise and we should not encourage this practice.

If you want to go that road, you can pipe your password before to call the script:
```echo "${ELEMENTAL2_GPG_PASS}" | ./release_elemental.sh```